src/Anonymization/Anonymizator.php

```diff
@@ -142,69 +167,167 @@ public function anonymize(
+                $this->output->writeLine(\sprintf('[%s]', $timer));
 
                 $this->addAnonymizerIdColumn($table);
 
                 if ($atOnce) {
-                    yield from $this->anonymizeTableAtOnce($table, $anonymizers);
-                } else {
-                    yield from $this->anonymizeTablePerColumn($table, $anonymizers);
+                    $this->anonymizeTableAtOnce($table, $anonymizers);
+                    $this->logger->info(
+                        'Table "{table}" anonymized at once. Targets were: "{targets}" ({timer}).',
+                        $context + ['timer' => $timer]
+                    );
+                }
+                else {
+                    $this->anonymizeTablePerColumn($table, $anonymizers);
                 }
-            } finally {
+            }
+            catch (\Throwable $e) {
+                $this->logger->error(
+                    'Exception caught when anonymizing "{table}" table: {error}. (Targets were: "{targets}").',
+                    $context + ['error' => $e->getMessage()]
+                );
+            }
+            finally {
                 $cleanTimer = $this->startTimer();
-                // Cleanup everything, even in case of any error.
-                yield "   - cleaning anonymizers...";
+                // Clean up everything, even in case of any error.
+                $this->output->write("- cleaning anonymizers...");
                 \array_walk($anonymizers, fn (AbstractAnonymizer $anonymizer) => $anonymizer->clean());
 
                 $this->removeAnonymizerIdColumn($table);
 
-                yield $this->printTimer($cleanTimer);
-                yield '   - total ' . $this->printTimer($initTimer);
+                $cleanTimer = $this->formatTimer($cleanTimer);
+                $this->output->writeLine(\sprintf('[%s]', $cleanTimer));
+                $this->output->writeLine("- total " . $this->formatTimer($initTimer));
+                $this->output->outdent();
+
+                $this->logger->info(
+                    'Clean-up performed after anonymizing "{table}" table ({timer}).',
+                    $context + ['timer' => $cleanTimer]
+                );
             }
 
             $count++;
         }
     }
 
     /**
-     * Forcefuly clean all left-over temporary tables.
-     *
-     * This method could be dangerous if you have table names whose name matches
-     * the temporary tables generated by this bundle, hence the dry run mode
-     * being the default.
+     * Forcefully clean all leftover temporary tables, columns and indexes.
+     */
+    public function clean(): void
+    {
+        $schemaManager = $this->connection->createSchemaManager();
+
+        foreach ($this->collectGarbage() as $item) {
+            switch ($item['type']) {
+                case 'table':
+                    $this->output->writeLine(\sprintf(
+                        "Dropping table: %s", $item['name']
```

Comment:
I don't see any reason to write it multiline here and below, but that's a matter of taste.

Reply:
I like this formatting so I keep it as such:

```
-------------------------------- Symfony console --------------------------------

Items that will be deleted
--------------------------

 Tables:

 * _db_tools_sample_65f1bbc82a22d
 * _db_tools_sample_65f1bbc87f93d

 Columns and indexes:

 * user.target_table_db_tools_id_idx
 * user._db_tools_id

 Are you sure you want to drop all these database items? (yes/no) [no]:
 > yes

Dropping items
--------------

Dropping table: _db_tools_sample_65f1bbc82a22d
Dropping table: _db_tools_sample_65f1bbc87f93d
Dropping index: user.target_table_db_tools_id_idx
Dropping column: user._db_tools_id

 [OK] Clean done!
```